phys2cvr/regressors.py

```diff
@@ -29,19 +29,19 @@
 
 def create_legendre(degree, length):
     """
-    Producesthe Legendre polynomials of order `degree`.
+    Produces Legendre polynomials of order `degree`.
```

Comment:
One thing that I didn't tell you is that the first line of the numpydoc has to be an imperative statement with a final dot.

```suggestion
    Produce Legendre polynomials of order `degree`.
```